Assets/Scripts/CameraShader.cs

```diff
@@ -47,7 +48,7 @@ public class CameraShader : MonoBehaviour {
     float fovEnd = active ? finalFOV : initialFOV;
     currentShaderCoroutine = StartCoroutine(TransitionShader(end, transitionTime, onComplete));
     currentVolumeCoroutine = StartCoroutine(TransitionVolume(end, transitionTime * 0.5f));
-    currentFOVCoroutine = StartCoroutine(TransitionFOV(fovEnd, transitionTime * 0.1875f));
+    currentFOVCoroutine = StartCoroutine(TransitionFOV(fovEnd, transitionTime * 0.25f));
```

Comment:
makes sense to make this a public global variable i think (could also be edited from unity editor)

Reply:
made it a private constant